origins_common/origins_common.module

```diff
@@ -85,3 +85,59 @@ function origins_common_form_entity_embed_dialog_alter(&$form, FormStateInterfac
   $form['attributes']['data-align']['#default_value'] = 'right';
   $form['attributes']['data-align']['#access'] = FALSE;
 }
+
+/**
+ * Implements hook_preprocess_table().
+ */
+function origins_taxonomy_access_preprocess_table(&$variables) {
+  // Add new 'Revision Number' column to the node revisions tab.
+  //
+  // Making this change in preprocess is not beautiful, but seems
+  // to be the only workable solution as the 'diff' module has
+  // already been enabled and has taken over the core route with
+  // its own controller. If we created our own controller to do
+  // this then we would have to call the contrib controller,
+  // which does not sound like a good idea.
+  if (\Drupal::routeMatch()->getRouteName() == 'entity.node.version_history') {
+    // Add 'Revision Number' column header.
+    $newhead = ['tag' => 'th', 'content' => ['#markup' => t('Revision Number')]];
```

Comment:
Something to be mindful of with labels and such like is consistency around Sentence case and Title case. Essentially, either is fine but keeping things consistent makes a big difference to the UX.

See https://uxplanet.org/why-letter-casing-is-important-to-consider-during-design-decisions-50402acd0a4e for details.

Personally, I'm all for sentence case as I find it much, much easier to read... but that's just me. Apple sees things differently here... but who listens to them!? 🤣

Reply:
I agree with you, I prefer sentence case too generally, have corrected this as suggested